core/templates/pages/topics-and-skills-dashboard-page/navbar/topics-and-skills-dashboard-navbar-breadcrumb.component.ts

```diff
@@ -13,21 +13,22 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controller for the navbar breadcrumb of the collection editor.
+ * @fileoverview Component for the navbar breadcrumb of the collection editor.
```

Comment:
Can you please fix the fileoverview?

Not `collection editor`, it should be `topics and skills dashboard`.